app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragment.java

```diff
@@ -980,6 +980,19 @@ public void onClick(View view) {
                 }
             });
             mDetailsOverviewRow.addAction(playButton);
+
+            if (mBaseItem.getIsFolderItem() || baseItem.getType() == BaseItemKind.MUSIC_ARTIST) {
```

Comment:
MUSIC_ARTIST can be removed here because music related stuff doesn't use the video player so this would bring up a player selection that won't be used.